src/main/java/seedu/recipe/model/recipe/Name.java

```diff
@@ -11,9 +11,13 @@
 
     public static final String MESSAGE_CONSTRAINTS =
             "Names should only contain alphanumeric characters, spaces, or special characters from "
-                    + "this set {&, %, (, ), -}. The name should also not be blank.";
+                    + "this set {&, %, (, ), -, /}. The name should also not be blank.";
 
-    public static final String VALIDATION_REGEX = "^[\\sA-Za-z0-9\\()&%-]+$+";
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "^[\\sA-Za-z0-9\\()&%-/]+$+";
```

Comment:
I think the validation regex here should be: "^[\sA-Za-z0-9'/\()&%-]+$+" (note: I added an apostrophe to the valid characters too)

When I tested the string: "Grandma's 100% mac & (string) cheese - for summer/winter" it returned the whole string when the above regex was used (expected). However, when I tested this string against the validation regex in the line that I am commenting on, they could not find any matches!

Reply:
Thanks!